src/commands/highlights.rs

```diff
@@ -48,7 +48,8 @@ pub async fn highlights_add(ctx: &client::Context, msg: &Message, args: Args) ->
         .send_message(&ctx, |m| {
             m.embed(|e| {
                 e.title("Highlight added");
-                e.description(format!("Notifying you whenever someone says `{}`", trigger))
+                e.description(format!("Notifying you whenever someone says `{}`", trigger));
+                e.footer(|e| e.text("This is more of a test to see if DMs work, than a confirmation of the highlight being set successfully. You should look if the bot returned any errors in the channel."))
```

Comment:
This message is kinda,... weird. idk how exactly to phrase it, but that's not it, chief

Reply:
yeah i get what you mean lol